epub33/explainers/EPUB33-explainer.md

```diff
@@ -0,0 +1,114 @@
+# EPUB 3.3 Explainer
+
+**Written**: 2021-02-22, **Updated**: 2021-02-22
+
+**EPUB 3.3** is a packaging mechanism for web content, designed for electronic books. 
+
+
+
+Spec references:
+* [EPUB 3.3 Core](https://w3c.github.io/epub-specs/epub33/core//), the specification of the file format
+* [EPUB 3.3 Reading Systems](https://w3c.github.io/epub-specs/epub33/rs/), the specification for user agents, known as EPUB Reading Systems. 
+* [EPUB Accessibility 1.1](https://w3c.github.io/epub-specs/epub33/a11y/)
+* [EPUB Accessibility Techniques 1.1](https://w3c.github.io/epub-specs/epub33/a11y-tech/)
+
```

Comment:
We may want to refer to the Overview document and refer to it as a first read for people looking at the spec.